tests/test_run_app.py

```diff
@@ -42,6 +42,16 @@
 )
 del _has_unix_domain_socks, _abstract_path_failed
 
+has_ipv6 = socket.has_ipv6
```

Comment:
`has_ipv6` should be upper-cased: it is a contant

Reply:
I've just copied it from another test [module](https://github.com/aio-libs/aiohttp/blob/master/tests/test_http_stream_writer.py#L9).

Reply:
But please make it upper cased anyway

Reply:
OK, done.